androidApp/src/main/java/com/radityalabs/qualitymatter/androidApp/ui/Theme.kt

```diff
@@ -0,0 +1,47 @@
+package com.radityalabs.qualitymatter.androidApp.ui
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+    primary = purple200,
+    primaryVariant = purple700,
+    secondary = teal200
+)
+
+private val LightColorPalette = lightColors(
+    primary = purple500,
+    primaryVariant = purple700,
+    secondary = teal200
+
+    /* Other default colors to override
+    background = Color.White,
+    surface = Color.White,
+    onPrimary = Color.White,
+    onSecondary = Color.Black,
+    onBackground = Color.Black,
+    onSurface = Color.Black,
+    */
+)
+
+@Composable
+fun AppTheme(
```

Comment:
would be possible to add `kdoc`?

Reply:
You mean this?
https://kotlinlang.org/docs/reference/kotlin-doc.html
Do we just need to add the comment?

Reply:
yes, similar to what we've done in internal library

Reply:
Cool done it https://github.com/radityagumay/QualityMatters/pull/1/commits/f6c077b72baa6c805f86e996246627102d42648e

Reply:
beautiful mas!